source/standards/programming-languages.html.md.erb

```diff
@@ -79,10 +79,24 @@ production.
 
 ### Languages we do not use for new projects
 
+#### Scala
+
 We used Scala in the early days of GDS. GOV.UK Licensing is the only remaining
 application written in Scala but we've found it hard to support because of a lack
 of skills in GDS. Do not use Scala for new projects.
 
+#### Go
+
+Go is no longer a core backend development language in GDS.
+
+The only Go service currently in production operation is the [GOV.UK
+Foundry](https://www.cloudfoundry.org/), which was used by GOV.UK PaaS.
+
+PaaS has now been [decommissioned][pass-decom].
```

Comment:
PaaS has not yet been completely decommissioned. While it's no longer available for general use, it's still being operated within GDS, and used by DBT.